lib/plugins/aws/deploy/compile/events/apiGateway/lib/apiKeys.js

```diff
@@ -39,9 +39,9 @@ module.exports = {
         _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Resources,
           newApiKeyObject);
 
-        // Add API Key to Outputs section
+        // Add Id of API Key to Outputs section (Note: This is the Id, not the value!)
         const newOutput = {
```

Comment:
don't we need to remove this now since we're using the API Keys API directly? the output is pretty useless now